wordcount.c

```diff
@@ -0,0 +1,30 @@
+//Ashank Kumar, reads a file and prints the words in it
```

Comment:
Comment your code so I can help pick out where you went wrong! I will not spend too much time trying to figure out your code when you could have commented throughout. Your code didn't output the correct number on the test file.